pkg/api/machine.go

```diff
@@ -57,6 +57,7 @@ type VolumeStatus struct {
 	Name   string      `json:"name,omitempty"`
 	Handle string      `json:"handle,omitempty"`
 	State  VolumeState `json:"state,omitempty"`
+	Size   int64       `json:"size,omitempty"`
```

Comment:
Can be size negative?

Reply:
The size can't be negative but I'm consistent with the other size datatypes.